lib/generators/shopify_app/app_proxy_controller/templates/app_proxy_controller.rb

```diff
@@ -0,0 +1,12 @@
+class AppProxyController < ApplicationController
+  if Rails.env.development? || Rails.env.test?
+   around_action :shopify_session
+  else
+   include ShopifyApp::AppProxyVerification
+  end
```

Comment:
the controller was named **app_proxy_controller** in sync with existing generator for home_controller. 
`rails generate controller shopify_app:app_proxy_controller`

The other shorter alternative to 'app_proxy_controller' can be made to be **proxy_controller**
`rails g controller shopify_app:proxy_controller`

Once again, thanks for the inputs. :)